cpp/program/play.h

```diff
@@ -317,7 +317,8 @@ class GameRunner {
     Logger& logger,
     std::vector<std::atomic<bool>*>& stopConditions,
     std::function<NNEvaluator*()> checkForNewNNEval,
-    std::function<void(const Board&, const BoardHistory&, Player, Loc, const std::vector<double>&, const std::vector<double>&, const std::vector<double>&, const Search*)> onEachMove
+    std::function<void(const Board&, const BoardHistory&, Player, Loc, const std::vector<double>&, const std::vector<double>&, const std::vector<double>&, const Search*)> onEachMove,
+    bool logOwnership
```

Comment:
Should be named `alwaysIncludeOwnership` as well to be consistent with the implementation, and which is semantically more accurate.

Reply:
have named it includeOwnership in config and alwaysIncludeOwnership as var, to be consistent with the analysis cfg